### PR TITLE
Add unit tests workflow

### PR DESCRIPTION
**Pull request by iulianpascalau**
- added unit tests github workflow
- added a new function in BLS multisigner to be used in tests

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -1,6 +1,6 @@
 module github.com/ElrondNetwork/elrond-go-crypto
 
-go 1.16
+go 1.13
```
Comment (iulianpascalau) [inline go.mod:3]: without this change the tests fail with the following error:
```
# github.com/herumi/bls-go-binary/bls
vendor/github.com/herumi/bls-go-binary/bls/bls.go:17:10: fatal error: mcl/bn_c384_256.h: No such file or directory
   17 | #include <mcl/bn_c384_256.h>
      |          ^~~~~~~~~~~~~~~~~~~
compilation terminated.
```

File: .github/workflows/pr-tests.yml [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+        id: go
+
+      - name: Check out code
+        uses: actions/checkout@v2
+
+      - name: Get dependencies
+        run: |
+          go get -v -t -d ./...
+      - name: Unit tests
+        run: make test
```
Comment (bogdan-rosianu) [inline .github/workflows/pr-tests.yml:28]: empty line

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,3 @@
+test:
+	@echo "  >  Running unit tests"
+	go test -cover -race -coverprofile=coverage.txt -covermode=atomic -v ./...
```
Comment (bogdan-rosianu) [inline Makefile:3]: empty line

File: signing/multisig/blsMultisig_test.go [hunk 4]
```diff
@@ -37,9 +40,21 @@ func genMultiSigParamsBLS(cnGrSize int, ownIndex uint16) (
...
 		pubKeyBytes, _ = pk.ToByteArray()
 		pubKeys = append(pubKeys, string(pubKeyBytes))
+		privKeys = append(privKeys, sk)
 	}
 
-	return privKey, pubKey, pubKeys, kg, llSigner
+	return privKey, pubKey, privKeys, pubKeys, kg, llSigner
+}
+
+func genMultiSigParamsBLS(consensusGroupSize int, ownIndex uint16) (
```
Comment (bogdan-rosianu) [inline signing/multisig/blsMultisig_test.go:49]: `generate` ?

**Review by bogdan-rosianu — CHANGES_REQUESTED** [review]

File: signing/multisig/blsMultisig_test.go (hunk 4 above)
Comment (raduchis) [reply signing/multisig/blsMultisig_test.go:49]: :+1:

File: .github/workflows/pr-tests.yml (hunk 2 above)
Comment (iulianpascalau) [reply .github/workflows/pr-tests.yml:28]: done

File: Makefile (hunk 3 above)
Comment (iulianpascalau) [reply Makefile:3]: done

File: signing/multisig/blsMultisig_test.go (hunk 4 above)
Comment (iulianpascalau) [reply signing/multisig/blsMultisig_test.go:49]: old code, changed `gen` -> `generate`

**Review by raduchis — COMMENTED** [review]
small changes needed

**Review by raduchis — APPROVED** [review]

**Review by bogdan-rosianu — APPROVED** [review]

**Pull request merged**